doc/nrf/links.txt

```diff
@@ -537,6 +537,21 @@
 .. _`Memfault: Collecting Device Metrics`: https://docs.memfault.com/docs/embedded/metrics-api
 .. _`Memfault: Error Tracking with Trace Events`: https://docs.memfault.com/docs/embedded/trace-events/
 
+.. ### Source: nrfcloud.com
+
+.. _`nRF Cloud`:
```

Comment:
I guess these empty items are just placeholders for the product name change?

Reply:
That's more of a question to @melwee01 , but the placement of tags is juts a method for having two different tags use the same link. In this case, `nRF Cloud` and `nRF Connect for Cloud` will both link to https://nrfcloud.com/, and so on for other pairs.

Reply:
Oh, that was new to me. Thanks for the hot tip Greg!